### PR TITLE
15172

### PR DESCRIPTION
**Pull request by arasevic**
|Field|Value|
|-----|-----|
| Redmine Ticket Number | [15172](https://devops.dchbx.org/redmine/issues/15172) |
| App-Dev Road Map # | n/a |
| Start Date | n/a |
| Due Date | n/a  |
| App-Dev Priority | UP-Disqueued |
| Development Story Points | 2 |
| Peer Review Story Points | 0.5 |
| Ticket Author | @arasevic |

### CODE REVIEW

File: app/controllers/employers/employer_profiles_controller.rb [hunk 1]
```diff
@@ -121,8 +121,22 @@ def show
       when 'accounts'
         collect_and_sort_invoices(params[:sort_order])
         @sort_order = params[:sort_order].nil? || params[:sort_order] == "ASC" ? "DESC" : "ASC"
+
+        #grab url for WellsFargoSSO and store in insance variable
+        email = (@employer_profile.staff_roles.first && @employer_profile.staff_roles.first.person.emails.first &&
+          @employer_profile.staff_roles.first.person.emails.first.address) || nil
+
+        if email.present?
+          w = WellsFargo::BillPay::SingleSignOn.new(@employer_profile.hbx_id, @employer_profile.hbx_id, @employer_profile.dba, email)
```
Comment (antonioschaffert) [inline app/controllers/employers/employer_profiles_controller.rb]: change w to wells_fargo_sso? This makes it easier to read

File: app/controllers/employers/employer_profiles_controller.rb [hunk 2]
```diff
@@ -121,8 +121,22 @@ def show
...
+        end
+
+        if w.present?
+          if w.token.present?
+            @wf_url = w.url
+          end
+        end
       when 'employees'
-        @current_plan_year = @employer_profile.show_plan_year
+        @current_plan_year = @employer_profile.ddshow_plan_year
```
Comment (antonioschaffert) [inline app/controllers/employers/employer_profiles_controller.rb]: is this a typo? ddshow_plan_year ?

File: app/views/employers/employer_profiles/my_account/accounts/_accounts.html.erb [hunk 3]
```diff
@@ -4,9 +4,9 @@
   <table class="table">
     <thead>
       <tr>
-        <th>Account Balance <i class="fa fa-info-circle" data-toggle="tooltip" data-placement="right" title="Account Balance includes payments and enrollment changes made since the latest invoice but have not yet been billed.  This may not be the total amount that is currently due."></i>
+        <th>Total Amount Due <i class="fa fa-info-circle" data-toggle="tooltip" data-placement="right" title="The amount you owe after any credits or payments made since your last invoice."></i>
         </th>
-        <th>Past Due Amount <i class="fa fa-info-circle" data-toggle="tooltip" data-placement="right" title="Past Due Amount includes premiums that were previously billed but were not paid by the due date. If premiums are not paid within 2 months of the due date, your group’s health coverage may be terminated."></i>
+        <th>Past Due Amount <i class="fa fa-info-circle" data-toggle="tooltip" data-placement="right" title="The amount you have not paid from prior invoices. If premiums are not paid within 2 months of the due date, your group’s coverage will be terminated."></i>
```
Comment (jayreddy519) [inline app/views/employers/employer_profiles/my_account/accounts/_accounts.html.erb]: Updated Text under Account balance & Past Due Amount.

File: app/views/employers/employer_profiles/my_account/accounts/_current_activity.html.erb [hunk 4]
```diff
@@ -11,18 +11,9 @@
...
-      <td>Remaining amount due from <%= @employer_profile_account.current_statement_date %> invoice
-        <i class="fa fa-info-circle" data-toggle="tooltip" data-placement="right" title="This amount is the remaining amount due from the last invoice balance due minus any advanced credits for enrollment changes, or payments made since the last invoice date. Invoices must be paid in full or else the group is at risk of going into arrears. Any payment or enrollment activity that occurs after the last invoice date will be reflected on the next invoice."></i>
-      </td>
-      <td>
-        <%= number_to_currency(@employer_profile_account.total_due) || "Data not available" %>
-      </td>
-    </tr>
     <!-- Payments -->
     <tr>
+      <td><%= @employer_profile_account.current_statement_date %></td>
```
Comment (jayreddy519) [inline app/views/employers/employer_profiles/my_account/accounts/_current_activity.html.erb]: Removed unused code. 
Added current statement date.

File: app/views/employers/employer_profiles/my_account/accounts/_current_activity.html.erb [hunk 5]
```diff
@@ -52,7 +43,7 @@
     <tr>
       <td class="adjustments-icon"><span class="pull-right glyphicon glyphicon-plus"></span></td>
       <td>Enrollment changes since <%= @employer_profile_account.current_statement_date %> invoice
-        <i class="fa fa-info-circle" data-toggle="tooltip" data-placement="right" title="Enrollment Changes are any type of enrollment change, including new enrollments, changes to existing enrollments, or terminations of coverage. Enrollment changes may occur due to a variety of reasons, including newly-eligible for coverage, special enrollment periods, or terminations of eligibility."></i>
+        <i class="fa fa-info-circle" data-toggle="tooltip" data-placement="right" title="Includes new enrollments, changes to existing enrollments, and terminations. Enrollment changes happen when you make a new employee eligible for coverage, when someone qualifies for a special enrollment period, or when someone is no longer eligible for coverage."></i>
```
Comment (jayreddy519) [inline app/views/employers/employer_profiles/my_account/accounts/_current_activity.html.erb]: Updated text on current activity view page.

File: app/views/employers/employer_profiles/my_account/accounts/_pay_my_bill.html.erb [hunk 6]
```diff
@@ -30,7 +28,7 @@
         <%=  Settings.contact_center.alt_phone_number %>, select "Small Business Employer" then "Make a Payment"
       </div>
     </li>
-    <li>Pay by Mail <i class="fa fa-info-circle" data-toggle="tooltip" data-placement="right" title="SLOWEST payment option- up to 2 weeks processing"></i>
+    <li>Pay by Mail <i class="fa fa-info-circle" data-toggle="tooltip" data-placement="right" title="Slowest payment option – takes up to 2 weeks to process."></i>
```
Comment (jayreddy519) [inline app/views/employers/employer_profiles/my_account/accounts/_pay_my_bill.html.erb]: Updated text.

File: app/views/employers/employer_profiles/my_account/accounts/_payment_history.html.erb [hunk 7]
```diff
@@ -1,15 +1,13 @@
...
     <p>
-      Allow up to 2-3 business days for online or phone payments to be reflected below.  Allow up to 2 weeks for mailed payments to be reflected below.  For fastest processing, click on
-      <%= link_to 'Pay My Bill', @wf_url, id: :sso_enabled, target: :_blank %> to make an online payment now.
+      Online and phone payments will appear 2-3 business days after you submit them. Mailed payments will appear within 2 weeks after you mail them. <%= link_to 'Pay My Bill', @wf_url, id: :sso_enabled, target: :_blank %> online is the fastest way to pay.
     </p>
   <% else %>
     <p>
-      Allow up to 2-3 business days for online or phone payments to be reflected below.  Allow up to 2 weeks for mailed payments to be reflected below.  For fastest processing, click on
-      <%= link_to 'Pay My Bill', @wf_url, id: :sso_disabled, target: :_blank %> to make an online payment now.
+      Online and phone payments will appear 2-3 business days after you submit them. Mailed payments will appear within 2 weeks after you mail them. <%= link_to 'Pay My Bill', @wf_url, id: :sso_disabled, target: :_blank %> online is the fastest way to pay.
```
Comment (jayreddy519) [inline app/views/employers/employer_profiles/my_account/accounts/_payment_history.html.erb]: Updated text under payment history.

**Pull request merged**